Files/Assets/terminal/terminal.json

```diff
@@ -5,7 +5,7 @@
     {
       "name": "CMD",
       "path": "cmd.exe",
-      "arguments": "/k \"cd /d {0} && title Command Prompt\"",
```

Comment:
This change will not update user's `terminal.json`. So will it work with old arguments?

Reply:
Yes, old arguments still work.